utils/yq/Makefile

```diff
@@ -0,0 +1,40 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=yq
+PKG_VERSION:=3.4.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/mikefarah/yq/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=73259f808d589d11ea7a18e4cd38a2e98b518a6c2c178d1ec57d9c5942277cb1
+
+PKG_MAINTAINER:=Tianling Shen <cnsztl@project-openwrt.eu.org>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+
+GO_PKG:=github.com/mikefarah/yq
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/yq
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=yq is a portable command-line YAML processor
+  URL:=https://mikefarah.gitbook.io/yq/
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Package/yq/description
+  The aim of the project is to be the jq or sed of yaml files.
+endef
+
+define Package/yq/install
+	$(INSTALL_DIR) $(1)/usr/bin/
+	$(INSTALL_BIN) $(GO_PKG_BUILD_BIN_DIR)/yq $(1)/usr/bin/
+endef
```

Comment:
`$(eval $(call GoBinPackage,yq))` should do the same as this, but it's fine to keep as-is.